coins.json

```diff
@@ -1249,9 +1249,9 @@
         "curve": "ed25519",
         "publicKeyType": "ed25519",
         "explorer": {
-            "url": "https://??",
-            "txPath": "/??",
-            "accountPath": "/??"
+            "url": "https://polkadot.subscan.io",
+            "txPath": "/extrinsic/",
+            "accountPath": "/account/"
```

Comment:
Polkatdot explorer update could be in separate PR.

Reply:
I know, I just saw the compiling warnings, but CI is too slow :(, don't want to create a trival pr for it